llpc/tool/llpcCompilationUtils.cpp

```diff
@@ -148,6 +148,20 @@ static ShaderStage sourceLangToShaderStage(SpvGenStage sourceLang) {
   }
 }
 
+// =====================================================================================================================
+// Disassemble SPIR-V binary file.
+//
+// @param binSize: Binary size.
+// @param code: Pointer to binary.
+// @param header: Extra info.
+static void disassembleSpirv(unsigned binSize, const void *code, const std::string &header) {
```

Comment:
nit: based on the uses, `Twine` would work fell for `header`

Reply:
Done.